tests/staking/PauseStaking.test.js

```diff
@@ -409,6 +452,19 @@ contract("Staking", (accounts) => {
 
             let vestingBalance = await staking.balanceOf(vesting.address);
             expect(vestingBalance).to.be.bignumber.equal(new BN(0));
+
+            /// should emit token withdrawn event for complete withdrawal
+            const end = vesting.endDate();
+            tx = await staking.cancelTeamVesting(
+                vesting.address,
+                root,
+                new BN(end.toString()).add(new BN(TWO_WEEKS))
+            );
+
+            expectEvent(tx, "TeamVestingCancelled", {
```

Comment:
this test's title is `should not allow governanceDirectWithdrawTokens when frozen` 
this is either error or should be extracted into another test

Reply:
updated the description in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/7f1c269ee40b1d2f3f864b07c574360839a9d270)